File: Wordle/LetterGuess.cs
using System;
namespace Wordle
{
	public enum LetterResult
	{
		Correct,
		Misplaced,
		Incorrect
	}

	public class LetterGuess
	{
        public char Letter { get; set; }

		public LetterResult LetterResult { get; set; }

        public LetterGuess(char letter)
		{
			Letter = letter;
			LetterResult = LetterResult.Incorrect;
		}
	}
}

File: pywordle/letter_guess.py
from enum import Enum


class LetterResult(Enum):
    correct = 1,
    misplaced = 2,
    incorrect = 3


class LetterGuess:
    def __init__(self, letter: str):
        self.letter = letter
        self.letter_result = LetterResult.incorrect

    def __str__(self):
        return f"{self.letter}: {self.letter_result.name}"
